docs/develop/03_workshop/04_attester/03_ctype.md

```diff
@@ -44,14 +44,12 @@ For example, a very basic CType for a driver's license could look like this:
 
 Let's have a look at these attributes.
 
-| Key                     | Value                                                                    |
-| ----------------------- | ------------------------------------------------------------------------ |
-| `schema` > `$id`        | The KILT id of this CType.                                               |
-| `schema` > `$schema`    | The JSON schema on which the CType is based.                             |
-| `schema` > `title`      | The title of the CType.                                                  |
-| `schema` > `properties` | The properties that a claim of type in `$schema` should have.            |
-| `owner`                 | The DID of the user who created this CType.                              |
-| `hash`                  | Most important attribute, the hash is the CType's **digital footprint**. |
+| Key                     | Value                                                                                                                                                               |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `schema` > `$id`        | The KILT id of this CType. It is the most important property as it represents the **digital footprint** of the CType.                                               |
+| `schema` > `$schema`    | The JSON schema on which the CType is based.                                                                                                                        |
+| `schema` > `title`      | The title of the CType.                                                                                                                                             |
```

Comment:
This feels broken

Reply:
It's only how it's laid out in the text format, but the generated HTML is ok.

Reply:
Damn i missed the `schema >` everywhere. I’ll open a PR